src/firmware-update.html

```diff
@@ -3,30 +3,50 @@
 description: Zigbee and Thread USB stick by the creators of Home Assistant
 ---
 <div class="banner">
-  <span class="secondary">Web-Flasher</span>
+  <span class="secondary">Documentation</span>
   <span>Firmware updates for Connect&nbsp;ZBT-1</span>
 </div>
 <div class="container">
   <div>
-    <h1>Firmware update</h1>
-    <p>A firmware upgrade is not required for the Home Assistant Connect&nbsp;ZBT-1 at this point. There are no known bugs with the firmware it ships with.</p>
-    <p>We're working on making firmware updates available via Home Assistant. You will receive updates automatically in the future.
+    <p>The Connect&nbsp;ZBT-1 ships with a full featured Zigbee firmware. <em>At this point, there are no known issues with the factory flashed firmware.</em></p>
```

Comment:
hm.. what do you mean by full featured: extensive, feature-rich, complete, full-fledged, fully developed, ...

Reply:
Yeah "complete", as it has all things needed for a Zigbee controller.